### ISSUE TITLE
aco_update wrong behavior on non-existent Controller folder

### ISSUE THREAD

**Issue by boindil** [cakephp/acl#63]
```
bin/cake acl_extras aco_update

Created Aco node: controllers/FOC/Authenticate
```

This is happening again and again. The only folder containing files in src is ```plugins/FOC/Authenticate/src/Auth```

Created entry in database looks like this:
```id, parent_id, model: NULL, foreign_key: NULL, alias: FOC\Authenticate, lft, rght: lft+1```

**Comment by rchavik** [comment]
Can you verity that the infinite loop is fixed with #83 merged?

**Comment by boindil** [comment]
Yep this is fixed :+1:
